src/Features/Lsif/Generator/Program.cs

```diff
@@ -86,14 +86,16 @@ public static Task Main(string[] args)
                 outputWriter = outputFile;
             }
 
-            using var logFile = log != null ? new StreamWriter(log) : TextWriter.Null;
+            using var logFile = log != null ? new StreamWriter(log) { AutoFlush = true } : TextWriter.Null;
```

Comment:
Worth a comment explaining why the autoflush?

Reply:
The explanation in the commit message is probably enough.